packages/date-picker/src/basic/time-spinner.vue

```diff
@@ -273,6 +277,15 @@
 
         this.modifyDateField(label, now);
         this.adjustSpinner(label, now);
+      },
+      amPm(hour) {
+        let shouldShowAmPm = this.amPmMode.toLowerCase() === 'a';
+        if (!shouldShowAmPm) return '';
+        let isCapital = this.amPmMode === 'A';
+        let content = ' ';
+        if (shouldShowAmPm) content = content.concat((hour < 12) ? 'am' : 'pm');
```

Comment:
Here `if (shouldShowAmPm)` is not necessary since the method returns if `!shouldShowAmPm`. And `content = content.concat((hour < 12) ? 'am' : 'pm');` can move to the previous line.

Reply:
Revised.